keyboards/Atlas_65/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+F_CPU = 16000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
```

Comment:
These are not needed. 
```suggestion
```

These are set by default in `builddefs/mcu_seleection.mk`